droidmediacamera.cpp

```diff
@@ -268,6 +270,11 @@ DroidMediaBufferQueue *droid_media_camera_get_buffer_queue (DroidMediaCamera *ca
   return camera->m_queue.get();
 }
 
+DroidMediaBufferQueue *droid_media_camera_get_video_buffer_queue (DroidMediaCamera *camera)
```

Comment:
video_buffer_queue is a little ambiguous, recording_buffer_queue might be better understood by people not familiar with the Android API.

Reply:
> It's looking good. It's unfortunate that clients are going to have to be able to fall back to using the recording frame callback if there is no video buffer queue though.

Unfortunately yes. This is why I want to develop a separate droidmedia based plugin to hide all these details and make the camera interface consistent. Perhaps it will come in handy somewhere else.

Reply:
I renamed `droid_media_camera_get_video_buffer_queue()` to `droid_media_camera_get_recording_buffer_queue()` and tested on a device.